project-fortis-pipeline/docs/development-faq.md

```diff
@@ -131,3 +131,10 @@ that pre-populate your Fortis site.
 
 There are several examples of tar.gz files to bootstrap sites in the [seed-data](https://github.com/CatalystCode/project-fortis/tree/master/project-fortis-pipeline/localdeploy/seed-data)
 directory which you can use as a template.
+
+## Common Errors
```

Comment:
Nit: the rest of the headings in this section are phrased like questions given that this is supposed to be a FAQ.

Also: "common errors" is not a very specific heading given that this entire document is common errors. This section could be called more specifically: "How do I get the UI to load?" or similar.